tests/integration/attendances.test.js

```diff
@@ -80,7 +121,7 @@ describe('ATTENDANCES ROUTES - POST /attendances', () => {
       expect(response.statusCode).toBe(404);
     });
 
-    it('should return 403 if trainee doesn\'t belong to a company linked with course', async () => {
+    it('should return 403 if trainee is not registered and doesn\'t currently belong to a company', async () => {
```

Comment:
Ici il manque l'info: la structure doit etre rattachée a la fomation. Que penses-tu de : should return 403 if trainee is not registered and does not currently belong to a company related to this course ?